### ISSUE TITLE
cancel_batch_order Parameter logic encoding error 

### ISSUE THREAD

**Issue by 1163849662** [Binance-docs/binance-futures-connector-python#15]
Revise：
def cancel_batch_order(self, symbol: str, orderIdList: list= None, origClientOrderIdList: list= None, **kwargs):
elif (orderIdList is not None):

run error：{"code":-1130,"msg":"Data sent for parameter 'orderIdList' is not valid."}

**Comment by andrea-c-binance** [comment]
@1163849662 thanks for raising the issue, we have fixed it, please pull the latest version, thanks!

**Comment by lrauldan** [comment]
Hello, this does not seem to be fixed in 3.1.1:

```
[2022-06-02 08:44:58,565: DEBUG/ForkPoolWorker-8] https://testnet.binancefuture.com:443 "DELETE //fapi/v1/batchOrders?symbol=BTCUSDT&origClientOrderIdList=61cb41a6-a0b9-43d1-93d1-2a9050ae61cd&origClientOrderIdList=94e24acf-c3ba-4045-b359-1c9d3b7d75fd&origClientOrderIdList=dbd364ff-cf33-43a1-be48-1155c536e2f0&timestamp=1654159497710&signature=XXX HTTP/1.1" 400 84
```

origClientOrderIdList is not encoded as a list.

**Comment by xiandong79** [comment]
yes. I am facing the same issue
